app/controllers/api/v0/users_controller.rb

```diff
@@ -0,0 +1,59 @@
+class Api::V0::UsersController < ApplicationController
+    
+    def create
+        user = User.new(user_create_params)
```

Comment:
We will decide whether or not we want to refactor this based on how easily we can convert to GoLang...but good job!